src/containers/Landing/landing.css

```diff
@@ -0,0 +1,88 @@
+.landingPAge {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center
+}
+
+.Illustration {
+    width: 100%;
+    height: 56vh;
+}
+
+.landingPage__title {
+    width: 170px;
+    font-family: Poppins;
+    font-weight: bold;
+    font-size: 18px;
+    color: #242424;
+}
+
+.landingPage__content {
+    width: 197px;
+    font-family: Source Sans Pro;
+    font-size: 14px;
+    line-height: 18px;
+    text-align: center;
+    color: rgba(0, 0, 0, 0.6);
+}
+
+.landingPage__signUpLink {
+    width: 125px;
+    height: 33px;
+    font-family: Source Sans Pro;
+    font-size: 14px;
+    line-height: 18px;
+    color: #E9663A;
+    padding-top: 19px;
+}
+
+.landingPage__homeLink {
+    width: 226px;
+    font-family: Open Sans;
+    font-size: 14px;
+    line-height: 19px;
+    align-items: center;
+    color: #242424;
+}
+
+.landingPage__logo {
+    width: 226px;
+    height: 24px;
+    font-family: Open Sans;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 19px;
+    align-items: center;
+    color: #E9663A;
+    padding-left: 5px;
+}
+
+.landingPage__button {
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    color: #FFFFFF;
+    width: 221px;
+    height: 40px;
+    border-radius: 4px;
+}
+
+.ant-btn {
+    border: none;
+}
+
+.ant-btn-primary {
+    color: #fff;
+    background-color: #E9663A;
+    border-color: #E9663A;
+    text-shadow: none;
+    box-shadow: none;
+}
+
+.ant-btn-primary:hover, .ant-btn-primary:focus {
+    color: #fff;
+    background-color: #E9663A;
+    border-color: #E9663A;
+}
```

Comment:
Look to line 24,34, 43, and 54 
```css
 font-size: 14px;
```
you can collect all the repeated CSS and put it at the top with each other

Reply:
also, this is repeated
```CSS 
font-family: Source Sans Pro;
font-family: Open Sans;
``` 

and i don't know we are using a lot of fonts !!!!